<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MRP_SoftwareLicenseWidget</class>
 <widget class="RB_Widget" name="MRP_SoftwareLicenseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxPaint">
     <property name="title">
      <string>Sofware License</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="RB_ToolButtonBar" name="tbbSoftwareLicense" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="RB_TableView" name="tvSoftwareLicense">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Sales Order</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>License Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="deStartDate">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="deEndDate">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leLicenseCode"/>
   </item>
   <item row="3" column="3">
    <widget class="RB_IdxLineEdit" name="ileContact"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Start Date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="RB_IdxLineEdit" name="ileCustomer"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>End Date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="RB_IdxLineEdit" name="ileSalesOrder"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Customer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Contact</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Comment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="leComment"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_IdxLineEdit</class>
   <extends>QLineEdit</extends>
   <header>rb_idxlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>RB_Widget</class>
   <extends>QWidget</extends>
   <header>rb_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RB_ToolButtonBar</class>
   <extends>QWidget</extends>
   <header>rb_toolbuttonbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RB_TableView</class>
   <extends>QTableView</extends>
   <header location="global">rb_tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tvSoftwareLicense</tabstop>
  <tabstop>leLicenseCode</tabstop>
  <tabstop>deStartDate</tabstop>
  <tabstop>deEndDate</tabstop>
  <tabstop>leComment</tabstop>
  <tabstop>ileCustomer</tabstop>
  <tabstop>ileSalesOrder</tabstop>
  <tabstop>ileContact</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
